main.go

```diff
@@ -57,10 +57,9 @@ import (
 // xref: https://github.com/cilium/cilium/issues/23604
 
 const (
-	originalMTU               = 1500
-	bpfProgram                = "bpf/nat64.o"
-	reconcilePeriod           = 5 * time.Minute
-	widestPodCIDRRangeAllowed = 112
```

Comment:
this was 112 before, because we want to fit it into the ipv4 range, so we don't need to do dynamic allocation